unified-runtime/source/adapters/level_zero/image.cpp

```diff
@@ -28,8 +28,9 @@ ur_result_t
 urBindlessImagesImageFreeExp(ur_context_handle_t /*hContext*/,
                              ur_device_handle_t /*hDevice*/,
                              ur_exp_image_mem_native_handle_t hImageMem) {
-  UR_CALL(ur::level_zero::urMemRelease(
-      reinterpret_cast<ur_mem_handle_t>(hImageMem)));
+  auto Native = reinterpret_cast<ur_bindless_mem_handle_t *>(hImageMem);
+  delete Native;
```

Comment:
Delete is fine here since `ur_bindless_mem_handle_t`'s destructor calls `zeDestroyImage`.

Reply:
https://oneapi-src.github.io/unified-runtime/core/api.html#bindless-images-experimental

Doesn't seem to be

Reply:
This problem has been identified and is being fixed in https://github.com/intel/llvm/pull/18353